subiquity/common/filesystem/tests/test_gaps.py

```diff
@@ -82,6 +82,46 @@ def test_two_gaps(self):
         self.assertEqual(g2, gaps.at_offset(d, 60 << 20))
 
 
+class TestInteresects(unittest.TestCase):
```

Comment:
I'd point out the typo if I wasn't asking you to change the name of this function anyway.

Reply:
Done